modules/rosa-about.adoc

```diff
@@ -6,7 +6,7 @@
 
 :_content-type: CONCEPT
 [id="rosa-about_{context}"]
-= About the rosa CLI
+= About the {product-title} (ROSA) CLI, `rosa`
```

Comment:
I think we just want "ROSA CLI" in the title, otherwise this is going to be very long. It's also generally a good idea to avoid punctuation in headings as they can render poorly in some situations.